src/thread.cr

```diff
@@ -42,12 +51,19 @@ class Thread
   def initialize
     @func = ->{}
     @th = LibC.pthread_self
-    @main_fiber = Fiber.new
 
     @@threads.push(self)
   end
 
+  def is_process_thread?
```

Comment:
Drop `is_` prefix please, there's already `?` at the end, which communicates it's a predicate method.

Reply:
Done.